l3build-arguments.lua

```diff
@@ -40,12 +40,13 @@ option_list =
       {
         desc  = "Sets the config(s) used for running tests",
         short = "c",
-        type  = "table"
+        type  = "table",
+        complete = "lua_file"
       },
     date =
       {
         desc  = "Sets the date to insert into sources",
-        type  = "string"
+        type  = "string",
```

Comment:
Why the stray comma here?

Reply:
It's because I am too tired to check the format at that time. Fixed.